react/features/settings/components/web/audio/AudioSettingsPopup.js

```diff
@@ -65,7 +65,7 @@ function AudioSettingsPopup({
 }: Props) {
     return (
         <div className = 'audio-preview'>
-            <InlineDialog
+            <Popover
                 content = { <AudioSettingsContent
```

Comment:
if possible create a function which returns the component, and reference the function as content